File: square.py
import time

NUMBER = 10 ** 6   # 1 million
TIMES = 1000

def sum_squares(n):
    result = 0
    for i in range(n):
        result += i ** 2

    my_max = n * n
    return result

def sum_squares_repeat(number, times):
    for i in range(times):
        result = sum_squares(number)
        if result != 333332833333500000:
            print("FAILED") 


print("Summing {} squares {} times".format(NUMBER, TIMES))

start = time.time()
sum_squares_repeat(NUMBER, TIMES)
end = time.time()

print("Took {:f}s".format(end - start))

File: go/square.go
package main

import (
	"fmt"
	"time"
)

const Number uint64 = 1e6 // 1 Billion

func sumSquares(max uint64) uint64 {
	var result uint64 = 0
	for i := uint64(0); i < max; i++ {
		result += i * i
	}
	my_total := max * max
	println(my_total)
	return result

}

func main() {

	fmt.Printf("Summing %d squares\n", Number)

	start := time.Now()
	result := sumSquares(Number)
	end := time.Now()
	fmt.Println(result)

	fmt.Printf("Took %v\n", end.Sub(start))

}
